gradle/dependencies.gradle

```diff
@@ -9,7 +9,7 @@ def versions = [
   'jetbrains': '16.0.2',
   'jline': '0.9.94',
   'kryo': '2.22',
-  'linkedin-calcite-core': '1.21.0.260',
+  'linkedin-calcite-core': '1.21.0.262',
```

Comment:
What went in Calcite `1.21.0.261`? Is it a safe change?

Reply:
`1.21.0.261`: https://github.com/linkedin/linkedin-calcite/pull/98
`1.21.0.262`: https://github.com/linkedin/linkedin-calcite/pull/99

Needed one release for checkstyle fix 😅